src/api/roles/coach/coach.py

```
@@ -675,6 +689,147 @@ def get_coach_earnings(
+    Includes Client, Account without password, and telemetry objects.
+    """
+
+    if acc.coach_id is None:
+        raise HTTPException(403, detail="Coach profile required")
+
+    relationships = db.exec(
+        select(ClientCoachRequest, ClientCoachRelationship)
+        .join(ClientCoachRelationship, ClientCoachRelationship.request_id == ClientCoachRequest.id)
+        .where(
+            ClientCoachRequest.coach_id == acc.coach_id,
+            ClientCoachRequest.is_accepted == True,
```

Ruff (0.15.12)

[error] Avoid equality comparisons to `True`; use `ClientCoachRequest.is_accepted:` for truth checks

Replace with `ClientCoachRequest.is_accepted`

(E712)